test/Runtime/POSIX/DirSeek.c

```diff
@@ -11,8 +11,8 @@
 
 // For this test really to work as intended it needs to be run in a
 // directory large enough to cause uclibc to do multiple getdents
-// calls (otherwise uclibc will handle the seeks itself). We should
-// create a bunch of files or something.
+// calls (otherwise uclibc will handle the seeks itself).
+// Therefore gentemp generates a directory with a specific amount of entries
```

Comment:
I'm confused about this comment, what does it refer to?

Reply:
This is a comment from the time this test was still failing from time to time. It was already fixed by introducing the `gentemp` which creates a certain amount of  files/directories that make sure that uclic is invoking a couple of `getdents` calls.
It is not really related with this PR but also not enough enough for an own commit or PR.